db/migrate/20210820032713_add_actived_to_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddActivedToUsers < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :actived, :boolean, default: true
+
+    Rake::Task["user:migrate_actived"].invoke
```

Comment:
@hengsokly why do we need this migration, as the default: true should be assigned the previous record to true.